testpilot/frontend/static-src/app/templates/landing-page.js

```diff
@@ -42,7 +42,8 @@ export default `
               </div>
               <div class="intro-text delayed-fade-in-up">
                 <h2 class="banner">
-                  <span data-l10n-id="landingIntroOne" class="block">Test out new features.</span>
+                  <span data-l10n-id="landingIntroLead" class="block lead-in">Go beyond&hellip;</span>
```

Comment:
we should be consistent, i always just use ... instead since I've never had an issue with them rendering